README.md

```diff
@@ -1,11 +1,6 @@
 <a name="readme-top"></a>
```

Comment:
- [ ] This is just a placeholder for you to create and include a license file. Kindly remove this note, as it's meant just for you. 👍

![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/5637f7f6-46b4-492e-aad0-64b95a45eae5)